config/duration.go

```diff
@@ -8,8 +8,14 @@ import (
 
 // Duration is just an alias for time.Duration that allows
 // serialization to YAML as well as JSON.
-type Duration struct {
-	time.Duration `validate:"required"`
+type Duration time.Duration
```

Comment:
What if we said `type Duration = time.Duration`, aka making it a type _alias_ instead of a full type declaration? This would allow all of `time.Duration`'s existing methods to carry over, so we wouldn't need the new `.GetDuration()` and `.SetDuration()` methods.

Reply:
I initially tried that, I wound up receiving this type of error and switched to the current implementation.
```
root@d6cba12ff7cf:/boulder# go run ./cmd/boulder boulder-ca --config ./test/config/ca.json 
Error validating config file "./test/config/ca.json" for command "boulder-ca": json: cannot unmarshal string into Go struct field .CA.HealthCheckInterval of type time.Duration
exit status 1
```

Reply:
Ah right, we can either carry over the original impl's methods (by using a type alias), or be able to declare our own methods (by using a type definition), but not both. And we have to be able to declare our own methods in order to implement `UnmarshalJSON`. :(

In that case, I'd say just rename these methods to `.Get()` and `.Set()` and it'll look a lot cleaner.